src/python/WMCore/Services/WMStatsServer/WMStatsServer.py

```diff
@@ -124,3 +124,14 @@ def getChildParentDatasetMap(self, requestType="StepChain", parentageResolved=Fa
                         for childDS in childParentDS["ChildDsets"]:
                             childParentMap[childDS] = childParentDS["ParentDset"]
         return childParentMap
+
+    def getProtectedLFNs(self):
+        """
+        _getProtectedLFNs_
+
+        A method to be used for fetching a list of all protected lfns from WMStatServer
+        :param unmergedOnly: If True return only the lfns from the unmerged area
```

Comment:
This docstring line is invalid. I would also suggest to remove the extra spacing from the line below.